services/core-web/src/components/Forms/projectSummaries/ProjectSummaryForm.js

```diff
@@ -0,0 +1,263 @@
+      projectSummaryPermitTypesHash,
+    } = props;
+    const transformedAuthorizationTypesHash = transformAuthorizationTypes(
+      projectSummaryAuthorizationTypes
+    );
+    // We need to make sure we only add parent authorization type labels once to the form
+    const parentHeadersAdded = [];
+    return (
+      <div id="authorizations-involved">
+        <Typography.Title level={3}>Authorizations involved</Typography.Title>
+        <p>
+          These are the authorizations the proponent believes may be needed for this project.
+          Additional authorizations may be required.
+        </p>
+        <br />
+        {authorizations.length > 0 &&
+          authorizations.map((a) => {
+            const parentCode =
+              transformedAuthorizationTypesHash[a.project_summary_authorization_type]?.parent?.code;
+            const parentHeaderAdded = parentHeadersAdded.includes(parentCode);
+            return (
+              <>
+                {!parentHeaderAdded && parentHeadersAdded.push(parentCode) ? (
+                  <h2>
+                    {
+                      transformedAuthorizationTypesHash[a.project_summary_authorization_type]
+                        ?.parent?.description
+                    }
+                  </h2>
+                ) : null}
+                <h4 className="padding-sm--bottom">
+                  {
+                    transformedAuthorizationTypesHash[a.project_summary_authorization_type]
+                      ?.description
+                  }
+                </h4>
+                <p className="bold padding-sm--bottom">Types of permits</p>
+                {a.project_summary_permit_type.map((pt) => {
+                  return <p className="padding-md--left">{projectSummaryPermitTypesHash[pt]}</p>;
+                })}
+                <br />
+                <p className="bold padding-sm--bottom">Existing permit numbers involved</p>
+                {a.existing_permits_authorizations.map((epa) => {
+                  return <p className="padding-md--left">{epa}</p>;
+                })}
+                <br />
+              </>
+            );
+          })}
+      </div>
+    );
+  };
+
+  const renderContacts = () => {
+    const {
+      projectSummary: { contacts },
+    } = props;
+    return (
+      <div id="project-contacts">
+        <Typography.Title level={4}>Project contacts</Typography.Title>
+        <h3>Proponent contacts</h3>
+        <p className="bold">Primary project contact</p>
+        <p>{contacts[0]?.name}</p>
+        {contacts[0]?.job_title && <p>{contacts[0]?.job_title}</p>}
+        {contacts[0]?.company_name && <p>{contacts[0]?.company_name}</p>}
+        <a href={`mailto: ${contacts[0]?.email}`}>{contacts[0]?.email}</a>
+        <div className="inline-flex">
+          <p>{contacts[0]?.phone_number}</p>
+          {contacts[0]?.phone_extension && <p> ({contacts[0]?.phone_extension})</p>}
+        </div>
+        <br />
```

Comment:
For the primary contact, we'll have to add some safety for if it's null. I think we should add a null view here, but also ensure the form fails if there is no primary contact (This could all happen in a follow-up) Thoughts?

Reply:
This is a little bit lazy admittedly, however the backend will never allow for 0 contacts so I wasn't sure the best way to manage this. I believe above I had a destructuring default that included the object so  that this reference would never hard fail, but I am open to suggestions.

Reply:
I think its okay for this PR - but lets create a followup as I was able to POST this am without any contacts.